tests/library/inspector/cli-codegen-3.spec.ts

```diff
@@ -597,20 +597,36 @@ await page.GetByLabel("Coun\\"try").ClickAsync();`);
       recorder.trustedClick({ button: 'right' }),
     ]);
     expect(message.text()).toBe('right-clicked');
-    expect(await page.evaluate('log')).toEqual([
-      // hover
-      'button: pointermove',
-      'button: mousemove',
-      // trusted right click
-      'button: pointerup',
-      'button: pointermove',
-      'button: mousemove',
-      'button: pointerdown',
-      'button: mousedown',
-      'button: contextmenu',
-      'menu: pointerup',
-      'menu: mouseup'
-    ]);
+    if (browserName === 'chromium' && platform === 'win32') {
+      expect(await page.evaluate('log')).toEqual([
+        // hover
+        'button: pointermove',
+        'button: mousemove',
+        // trusted right click
+        'button: pointermove',
+        'button: mousemove',
+        'button: pointerdown',
+        'button: mousedown',
+        'button: pointerup',
+        'button: mouseup',
+        'button: contextmenu',
+      ]);
+    } else {
+      expect(await page.evaluate('log')).toEqual([
+        // hover
+        'button: pointermove',
+        'button: mousemove',
+        // trusted right click
+        'button: pointerup',
```

Comment:
Do you understand why is there a `pointerup` at the start of the right-click?

Reply:
pointer events were a fun way to try to describe how the world works for non mouse based inputs... https://w3c.github.io/pointerevents/#the-pointerup-event

Making all the events compatible/with pages is such a fun mess :) (signed, ex-(mobile)-browser/standards-dev)